app/controllers/offices_controller.rb

```diff
@@ -80,27 +75,17 @@ def office_params
   end
 
   def search_and_sort
-    if params[:query].present? && params[:order].present?
```

Comment:
This url is not being generated by our current UI design.
The integration of elasticsearch might be worth exploring in the future for more complex filtering.